{{cookiecutter.project_name|replace(" ", "")}}/Dockerfile

```diff
@@ -24,6 +24,7 @@ LABEL org.opencontainers.image.licenses="{{ cookiecutter.license }}"
 # hadolint ignore=DL3008,DL3013
 RUN apt-get update \
  && apt-get install -y --no-install-recommends software-properties-common \
+                                               gpg-agent \
```

Comment:
This was why it was previously failing to install deadsnakes/ppa on ubuntu 22.04